### DISCUSSION TITLE
New default model version

### DISCUSSION THREAD

**Discussion by rubenmejiac** [lllyasviel/Fooocus#1053] [General]
Just want to let the community know that KandooAI just released [Juggernaut XL version 7 + RunDiffusion](https://civitai.com/models/133005/juggernaut-xl), the new version of the default model used in Fooocus.
Please comment if you try it out.

**Comment by xhoxye** [comment]
The anime model has also been updated, and of course you can also download and use it yourself.
https://civitai.com/models/119012
blue_pencil-XL 1.0

**Reply by mashb1t** [reply]
@xhoxye fyi blue_pencil-XL V3 has just been published, same link: https://civitai.com/models/119012
